chapter_optimization/adagrad.md

```diff
@@ -139,6 +147,20 @@ def adagrad(params, states, hyperparams):
         p.grad.data.zero_()
 ```
 
+```{.python .input}
+#@tab tensorflow
+def init_adagrad_states(feature_dim):
+    s_w = tf.Variable(d2l.zeros((feature_dim, 1)))
+    s_b = tf.Variable(d2l.zeros(1))
+    return (s_w, s_b)
```

Comment:
The parenthesis is redundant

Reply:
@terrytangyuan Are you talking about this ```(s_w, s_b)``` parenthesis. If so then this parenthesis is used in both mxnet and pytorch.

Reply:
Oh I see then let's keep it consistent for now and polish later.